python/ray/autoscaler/_private/kubernetes/config.py

```diff
@@ -100,11 +100,16 @@ def get_resource(container_resources, resource_name):
 
 
 def _get_resource(container_resources, resource_name, field_name):
```

Comment:
Can you add a docstring describing the intent behind this implementation? Thanks!

Reply:
Yeah -- will add more comments.